frontend/src/index.css

```diff
@@ -7,7 +7,9 @@
     --toast-blue-1: rgb(3, 100, 255);
     --toast-blue-2: rgb(48, 98, 178);
     --toast-blue-2-transparent: rgb(48, 98, 178, 0.2);
-    --toast-purple: rgb(118, 26, 190);
+    --toast-purple-1: rgb(184, 117, 236);
+    --toast-purple-2: rgb(118, 26, 190);
+    --toast-purple-3: rgb(67, 9, 112);
```

Comment:
3 should be the lightest of the purples and 1 should be the darkest
(following the pattern of our neutrals) (the blues are backwards so that should be fixed later in a different story too)